modules/openfast-library/src/FAST_Types.f90

```diff
@@ -763,7 +763,7 @@ MODULE FAST_Types
     CHARACTER(1024)  :: RootName      !< Root name of FAST output files (overrides normal operation) [-]
     INTEGER(IntKi)  :: NumActForcePtsBlade      !< number of actuator line force points in blade [-]
     INTEGER(IntKi)  :: NumActForcePtsTower      !< number of actuator line force points in tower [-]
-    INTEGER(IntKi)  :: NodeClusterType      !< Node clustering for actuator line (0 - Uniform, 1 - Non-uniform clustered towards tip) [-]
+    LOGICAL  :: NodeClusterType      !< Node clustering for actuator line (0 - Uniform, 1 - Non-uniform clustered towards tip) [-]
```

Comment:
I'm wondering if this is from a bad merge. This seems to have been changed to an integer on July 14 (from @andrew-platt), but now it's a logical again. Could you regenerate all of the types files and see if that fixes this issue?